samza-yarn/src/main/java/org/apache/samza/job/yarn/FileSystemImplConfig.java

```diff
@@ -20,11 +20,7 @@
 
 import java.util.ArrayList;
 import java.util.List;
-import java.util.Set;
-import org.apache.commons.lang.StringUtils;
 import org.apache.samza.config.Config;
-import org.slf4j.Logger;
```

Comment:
This is mostly for my own understanding - don't we already have an "unused imports" check during build?
Or was the check somehow not being applied to these files?

Reply:
Our checkstyle does have an unused imports check, but these files never had checkstyle enabled.